src/components/QuicklinksCard.js

```diff
@@ -1,19 +1,26 @@
 import React from 'react';
 import { A, UL, LI, H2 } from './Typography';
-import { Card } from '../components/Common.js'
 
-// TODO: Create quicklinks card component
+import styled from 'styled-components'
+import { CardLike } from '../components/Common.js'
+
+const QuicklinksCard = styled.div`
+  ${CardLike};
+  margin: 0.5em 0;
+`
+
 export default ({ title, links }) => {
   return (
-    <Card>
+    <QuicklinksCard>
       <H2>{title}</H2>
       <UL>
         {
           links.map((link) =>
-            <LI><A href={link.href}>{link.label}</A></LI>
+            // TODO: "key" prop must be unique, else error in console (nbd imo? -AC)
```

Comment:
Hopefully `link.label` is unique for each link, I think we can remove this comment

Reply:
okay sounds good! i added this bc there's no guard anywhere for unique hrefs but i'm sure it'll be fine in practice ;)

Reply:
Were you getting console warnings/errors ?

Reply:
oh i got console errors with the dummy links because there were two google.coms but not anymore since we'll be using links from firebase from now on